tensorflow/core/common_runtime/function.cc

```diff
@@ -169,9 +169,15 @@ class FunctionLibraryRuntimeOverlay : public FunctionLibraryRuntime {
   Status RunSync(Options opts, Handle handle, gtl::ArraySlice<Tensor> args,
                  std::vector<Tensor>* rets) override;
 
+  Status RunSync(Options opts, Handle handle, gtl::ArraySlice<Tensor> args,
```

Comment:
What is the benefit of passing the callback into the method as opposed to invoking it when the method returns?

Reply:
I agree that it is not necessary. I have removed the callback and called the processing directly.